ansible/roles/http_proxy/tasks/main.yml

```diff
@@ -49,9 +49,8 @@
     ansible_ssh_pipelining: true
   ignore_errors: True
   with_items:
-    - {'user': 'ansible', 'home': '/home/ansibale' }
+    - {'user': 'ansible', 'home': '/home/ansible' }
     - {'user': 'root', 'home': '/root' }
-    - {'user': '{{ cchq_user }}' , 'home': '{{ code_home }}'}
```

Comment:
Removing this line because it's not needed and it was erroring. this tried to become cchq and then copy a file from /tmp/ansible which it doesn't have access to.

This doesn't stop bower from working because it uses the machines general http/https proxy